pex/interpreter.py

```diff
@@ -294,6 +294,13 @@ def iter_supported_platforms(self):
         # type: () -> Iterator[Platform]
         """All platforms supported by the associated interpreter ordered from most specific to
         least."""
+        yield Platform(
```

Comment:
I'm unable to grok whether this could duplicate one of the other platforms yielded below, and whether this would matter if so?

Reply:
It can't, although it wouldn't matter. This platform gets a 3 component full Python version unlike all the rest which are derived from wheel tags which only ever have a 2 component version.